x-pack/plugins/apm/public/components/alerting/ui_components/apm_rule_params_container/index.tsx

```diff
@@ -33,6 +56,10 @@ export function ApmRuleParamsContainer(props: Props) {
   }, []);
   return (
     <>
+      {showMinimumWindowSizeWarning && minimumWindowSize && (
```

Comment:
Looks like `useShowMinimumWindowSize` already checks if `minimumWindowSize` is defined and if not it return false so I don't think you need these two checks.

```suggestion
      {showMinimumWindowSizeWarning  && (
```

Reply:
afair this is needed to satisfy Typescript because `minimumWindowSize` is optional. I'll see what I can do though